### DISCUSSION TITLE
Can i upload multiple pdf's and create a table that indicates when to search in each pdf for answers to the chatbot querys?

### DISCUSSION THREAD

**Discussion by anahguardiet** [FlowiseAI/Flowise#2056] [Q&A]
E.g. i want to make a sustainability advisor, so i want to use docs that are sector specific, so i want to upload a bunch of docs but then also have a table that uses tags and columns to make sense of the category where these docs belong to.

**Comment by toi500** [comment]
Do u mean something like this?

![Screenshot 2024-03-31 174626](https://github.com/FlowiseAI/Flowise/assets/138339291/5947fd67-9a28-49ed-a5dd-5785b7a1dae1)

No table here but you can set a flow where, using the MR QA Chain, you can tell the model when to use a set of documents or topic.

**Comment by anahguardiet** [comment]
what about using folder with files? Seems to be an option in document loaders but theres not documentation on it
